src/main/scala/EnsimePlugin.scala

```diff
@@ -30,6 +30,9 @@ object Imports {
     val additionalCompilerArgs = TaskKey[Seq[String]](
       "Additional arguments for the presentation compiler."
     )
+    val javaFlags = TaskKey[Seq[String]](
```

Comment:
you've made this as a `Task` rather than a `Setting`, is that intentional? Settings are fixed, whereas Tasks are calculated.